### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Initial stab at more docs. Small intro, and docs for Stores. Let me know what y'all think!

### CODE REVIEW

File: content/reference/stores.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+      assert.equal(typeof heading, 'string')   // 3.
+      state.heading = heading                  // 4.
+      emitter.emit('render')
+    })
+  })
+})
+```
+
+1. Import the `'assert'` package.
+2. We define a deafult value for our `'header'` (we're doing something with a
```
Comment (goto-bus-stop) [inline content/reference/stores.md]: s/deafult/default

**Comment by goto-bus-stop** [comment]
lookin good on initial skim! 👏👏 sorry for only commenting a typo 😢

**Comment by yoshuawuyts** [comment]
@goto-bus-stop thanks for reviewing! If a typo is the worst we're doing, that's probably not half bad :grin:

File: content/reference/stores.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+   store our tweets in.
+3. When the `tweets:add` event is emitted, we'll push the new tweet into our
+   list of tweets and re-render. In a real app this logic we'd probably do a
+   lot more logic here to construct the tweet.
+4. We're creating a store for our user user. Usually this would probably be
+   stored as `stores/user.js`.
+5. We'll need to store two values: the username, and whether or not we're
+   logged in.
+6. When `user:login` is called, we set the username and set us as logged in.
+7. When `usser:logout` is called, we reset all values and log us back out.
```
Comment (goto-bus-stop) [inline content/reference/stores.md]: s/usser/user

**Comment by goto-bus-stop** [comment]
stores also take a third `app` parameter, should this doc mention something about that too? it might be more relevant for people writing reusable choo modules than apps. it's quite situational so i could also imagine it just being in the api reference or in some sort of 'writing reusable modules' doc.

**Comment by yoshuawuyts** [comment]
@goto-bus-stop oh yeah, good one! — I think adding it might actually be useful as its own lil heading :D

**Comment by yoshuawuyts** [comment]
Done! :grin:

**Comment by yoshuawuyts** [comment]
Alright, merging!

**Pull request merged**